apps/space/core/components/issues/filters/applied-filters/root.tsx

```diff
@@ -39,13 +39,21 @@ export const IssueAppliedFilters: FC<TIssueAppliedFilters> = observer((props) =>
       const priority = key === "priority" ? value : (issueFilters?.filters?.priority ?? []);
       const labels = key === "labels" ? value : (issueFilters?.filters?.labels ?? []);
 
-      let params: any = { board: activeLayout || "list" };
-      if (priority.length > 0) params = { ...params, priority: priority.join(",") };
-      if (state.length > 0) params = { ...params, states: state.join(",") };
-      if (labels.length > 0) params = { ...params, labels: labels.join(",") };
-      params = new URLSearchParams(params).toString();
+      const params: {
+        board: TIssueLayout | string;
+        priority?: string;
+        states?: string;
+        labels?: string;
+      } = {
+        board: activeLayout || "list",
+      };
 
-      router.push(`/issues/${anchor}?${params}`);
+      if (priority.length > 0) params.priority = priority.join(",");
+      if (state.length > 0) params.states = state.join(",");
+      if (labels.length > 0) params.labels = labels.join(",");
+
+      const qs = new URLSearchParams(params).toString();
+      router.push(`/issues/${anchor}?${qs}`);
```

Comment:
I think router.push has a second form where you could push an object in which case you could just push the url params and path and not have to name it